packages/serverpod/lib/src/authentication/authentication_info.dart

```diff
@@ -1,20 +1,20 @@
 import 'scope.dart';
 import '../server/session.dart';
 
-/// Returns authentication information for a given [Session] and [key] or null
+/// Returns authentication information for a given [Session] and [token] or null
 /// if the key is invalid.
 typedef AuthenticationHandler = Future<AuthenticationInfo?> Function(
-    Session session, String key);
+    Session session, String token);
```

Comment:
Suggestion(Non blocking NIT): Formatting fix.

```suggestion
    Session session, String token,);
```